client/src/components/content/PropertiesPanel/PropertiesPanel.jsx

```diff
@@ -182,6 +183,18 @@ const PropertiesPanel = ({ modeler }) => {
     );
   };
 
+  /**
+   * @description Gets called when a new app was selected in the dropwdown in the sidebar. Resets the task to default value
```

Comment:
What do you think about writing `applications`instead of `app`? (like we are doing it with `Bot` <> `Robot`) 
I would appreciate that and, in the hope you agree, I have put it in the coding standards in already.

Reply:
+1